tests/test_reports.py

```diff
@@ -167,10 +187,17 @@ def test_get_applications_report_query_param(
         == "eoi_reference,organisation_name,organisation_type,asset_type,"
         "geography,capital,revenue"
     )
-    for line in line2, line3:
-        field1, field2, _, _, field5, _, _ = line.split(",")
-        # could also do this to ignore all fields after 5.
-        # field1, field2, _, _, field5, *_ = line.split(",")
-        assert field1 == "Test Reference Number"
-        assert field2.startswith("Test Org Name ")
-        assert field5 == "W1A 1AA"
+
+    field1, field2, _, _, field5, _, _ = line2.split(",")
+    # could also do this to ignore all fields after 5.
```

Comment:
I think this got committed as part of discussion on my previous PR so can be removed :) Here and line 199